### DISCUSSION TITLE
Application Breaking

### DISCUSSION THREAD

**Discussion by nsanzimanagilbert** [opencrvs/opencrvs-core#4084] [Technical Q&A] [answered]
Hello team,
I have been running opencrvs-core along with my country configurations but somewhere the application failed and indicated errors in files that were never touched or altered. 
I tried finding out what the issue is but couldn't get on.  See screenshot. 
Kindly guide!
![image](https://user-images.githubusercontent.com/38226948/195540032-d0dfa73a-691f-4a55-b404-06d5a1f96a6b.png)

**Comment by Zangetsu101** [comment]
Hi @nsanzimanagilbert,
Really glad to hear that you are trying out OpenCRVS. In order for us to help you out, I wonder if you could provide some more information:

1. Which version of the opencrvs-core and farajaland are you currently running?
2. The steps you took (or the commands you ran) that lead you to this error.

**Reply by nsanzimanagilbert** [reply]
1. I am running version 1.1.0 
2. `yarn dev` to start to start the development 
In addition to this, shared is the error being indicated in the shown `tsx` file
![image](https://user-images.githubusercontent.com/38226948/195545105-944c0acb-3507-4bdb-b9a7-1517286e6cdd.png)

**Comment by euanmillar** [comment]
Hi @nsanzimanagilbert 

Did you run the installation steps precisely as they are described in the documentation [here](https://documentation.opencrvs.org/setup/3.-installation/3.1-set-up-a-development-environment) using the ```bash setup.sh``` command?  That command compiles all components for you.

I ran the setup script yesterday and did not encounter the error you raised.  I recommend that you try deleting your checked out repositories and try running the installation process all over again.  Perhaps you may have missed a step.

Are you running opencrvs-core from the master branch?
Also can you tell us the Node version you are running?
Finally are you running Ubuntu, Mac or Windows?  OpenCRVS only works on Linux operating systems.

**Reply by euanmillar** [reply]
You could also try deleting all node_modules folders, running ```yarn cache clean``` and ```yarn --force``` in both core and country configuration

**Reply by nsanzimanagilbert** [reply]
Yes I ran the installation as described in the documentation. 

Yes I am running opencrvs-core in the master branch.
I am using node version 14.18.1 and installed it using nvm.
I am running Ubuntu.

I have considered running the `git clone https://github.com/opencrvs/opencrvs-core.git`. I have been trying to figure out the errors for the past many hours. 

Thanks for the replies team!

**Comment by euanmillar** [comment, marked as answer]
@nsanzimanagilbert I believe our colleague @Zangetsu101 met with you and that you experienced this issue because you made changes to core and had introduced a bug.  There was no issues with the core code.

**Reply by nsanzimanagilbert** [reply]
Hi @euanmillar!
Very true, had introduced in some bugs during my modifications. I had a good engagement with @Zangetsu101 and everything is running super fine. Many thanks for the due feedback and support!
